README.md

```diff
@@ -26,10 +26,13 @@ https://soundscape-community.github.io/soundscape-web-client/replay_gpx.html
 To use a location other than what's reported by your device, include the latitude, longitude, and compass heading (degrees from north) in the URL, e.g. https://soundscape-community.github.io/soundscape-web-client/?lon=-77.006156&lat=38.897600&heading=0.0 will place you near Union Station in Washington, D.C.
 
 ### Running the app on your own machine
-1. Install dependencies and start the server
-    ```
-    $ npm install && npm start
-    ```
+```
+<!-- npm install -g @ionic/cli -->
+npm ci
+npm run build
+npm start
+ngrok http http://127.0.0.1:8080/
```

Comment:
This last step is only required to access from another device, right? If so, I'd separate this out and explain that after the previous step, the app is already running and accessible from a local browser.

I also think you may have unwittingly proposed a solution to #20 -- if so, please comment there :)

Reply:
At least in android, if you want to test it on a real device , it needs to be HTTPS, otherwise android will not give access to few things .... So an easy way is to use ngrok (you need at least  a free account). Yes, you are correct, you can access to it in your laptop to you localhost:8080 and even from your phone using your laptop LAN IP, but it will not give HTTPS... Not sure about the #20 but it should not be hard...